tpkg/credns-setup/multitail.sh.in

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+# When this exits, exit all back ground process also.
+trap 'echo killing: $(jobs -p); kill $(jobs -p)' 0
+
+# iterate through the each given file names,
+for file in "$@"
+do
+	# show tails of each in background.
+        DIRNAME=`dirname $file`
+        BASENAME=`basename $DIRNAME`
+	stdbuf -i 0 -o 0 -e 0 tail -f $file \
+      | stdbuf -i 0 -o 0 -e 0 sed "s/nsd\[/$BASENAME[/g" &
```

Comment:
stdbuf mode `-o L -e L` is better on the second `sed` line, because the logs are line based, and that makes the output line-by-line separated from the different servers, like a syslog output would be.  For the test maybe not that relevant as a bugfix, but for multitail used elsewhere that would be a good fix, I think, unless it is used in a non-line oriented data flow.

Reply:
Done